tv-shows-app/src/components/features/shows/ReviewForm/ReviewForm.tsx

```diff
@@ -0,0 +1,108 @@
+import { StarRating } from "@/components/shared/StarRating/StarRating";
+import { IReview } from "@/typings/Reviews.type";
+import { Button, Flex, FormControl, FormErrorMessage, Heading, Input, Text } from "@chakra-ui/react";
+import { useState } from "react";
+
+interface IReviewsFormProps {
+	onAdd: (review: IReview) => void;
+}
+
+const validationInitialState = {
+	commentIsError: false,
+	ratingIsError: false,
+};
+
+export const ReviewsForm = ({onAdd}: IReviewsFormProps) => {
+	const [validation, setValidation] = useState(validationInitialState);
+	const [ratingValue, setRatingValue] = useState(0);
+
+	const onClickHandler = () => {
+		const comment = document.getElementById('js-comment') as HTMLInputElement;
+
+		const newValidation = {}
+
+		if (comment.value === '') {
+			newValidation.commentIsError = true;
+		}
+
+		if (!ratingValue) {
+			newValidation.ratingIsError = true;
+		}
+
+		const validationCheck = Object.values(newValidation).some(value => value === true);
+
+		if (validationCheck) {
+			setValidation(newValidation);
+			return;
+		}
+
+		const newReview: IReview = {
+			comment: comment.value,
+			rating: ratingValue,
+		};
+
+		onAdd(newReview);
+
+		comment.value = '';
+		setRatingValue(0);
+		setValidation(validationInitialState);
+	};
+
+	return (
+		<Flex
```

Comment:
Ako je komponenta forma, onda bi ovdje radje dodao jos `as="form"` i hendlao `onSubmit` event, i dolje na buttonu maknio click handler i dodao `type="submit"`. Semantički je točnije (a tako ćemo raditi s formama ubuduće)